tests/gateway/test_gateway_config_parsing.py

```diff
@@ -71,40 +72,16 @@ def test_api_key_parsing_env(tmp_path):
 
     assert _resolve_api_key_from_input(str(conf_path)) == file_key
 
-    del os.environ["KEY_AS_ENV"]
-
-
-def test_api_key_parsing_file(tmp_path):
-    key_path = tmp_path.joinpath("api.key")
-    config = [
-        {
-            "name": "claude-chat",
-            "type": "llm/v1/chat",
-            "model": {
-                "name": "claude-v1",
-                "provider": "anthropic",
-                "config": {
-                    "anthropic_api_key": str(key_path),
-                },
-            },
-        },
-    ]
-
-    key_path.write_text("abc")
-    config_path = tmp_path.joinpath("config.yaml")
```

Comment:
Looks like you're reverting my `tmp_path` change.

Reply:
Sorry about that @harupy ! Pushed fixes